flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/LeaderChangeClusterComponentsTest.java

```diff
@@ -68,8 +69,8 @@ public static void setupClass() throws Exception  {
 
 		miniCluster = new TestingMiniCluster(
 			new MiniClusterConfiguration.Builder()
+				.setNumTaskManagers(NUM_TMS)
 				.setNumSlotsPerTaskManager(SLOTS_PER_TM)
-				.setNumSlotsPerTaskManager(NUM_TMS)
```

Comment:
@tillrohrmann , the bug is this. I fixed it.

Reply:
Good catch. Thanks for fixing it.